packages/one-app-bundler/webpack/createResolver.js

```diff
@@ -32,5 +32,7 @@ module.exports = function createResolver({ mainFields, resolveToContext = false
     resolveToContext,
   });
 
-  return (request) => enhancedResolver.resolveSync({}, __dirname, request);
+  const trailingSlash = (process.platform === 'win32') ? '\\' : '/';
+  // if resolveToContext add a trailing slash to indicate the value is a folder rather than  a file
+  return (request) => `${enhancedResolver.resolveSync({}, __dirname, request)}${resolveToContext ? trailingSlash : ''}`;
```

Comment:
Does doing `path.normalize` not get us the same thing?

Reply:
Or [`path.sep`](https://nodejs.org/dist/latest-v12.x/docs/api/path.html#path_path_sep)?

Reply:
nope... tried that dependency and it doesn't add the trailing slash unfortunately

Reply:
@Francois-Esquire there is not `path` in this context, it is a plain string returned by `enhanced-resolve` otherwise `path.sep` would work